public/assets/styles/mss.css

```diff
@@ -67,6 +67,10 @@ section:last-of-type {
   background: #fff;
 }
 
+.zone-principale {
+  width: 700px;
```

Comment:
Ce `700px` est couplé au `1200px` des marges fixes. Il vaut mieux écrire `min-width: 0;`.

(cf. https://dev.to/timhecker/grid-cell-issue-with-white-space-nowrap--text-overflow-ellipsis-52g6)